modules/custom/az_select_menu/az_select_menu.module

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @file
+ * Provides configurable blocks of menu links.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function az_select_menu_theme() {
+  return [
+    'az_select_menu' => [
+      'base hook' => 'az_select_menu',
```

Comment:
Shouldn't this be `menu_block` or some other theme hook?

Reply:
menu_block uses `menu`
but I couldn't make my module find my theme template without it.